rest-api/test/unit_test/offline_test/biobank_samples_pipeline_mysql_test.py

```diff
@@ -390,7 +410,7 @@ def test_reconciliation_query(self):
 
     # orders/samples where something went wrong; don't include orders/samples from more than 7
     # days ago, or where 24 hours hasn't elapsed yet.
-    exporter.assertRowCount(missing, 5)
+    exporter.assertRowCount(missing, 7)
```

Comment:
Assert specifically the contents of the missing rows below.

I would have thought this would go up by 3, for each of the finalized tests in p_unconfirmed_samples_3.

Reply:
Sorry, I realized that p_unconfirmed_samples_3 is inserted at order_time, too... so I would have thought that *none* of these would show up in missing, as it hasn't been 36 hours yet.

Reply:
correct, fixed insert.